frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -0,0 +1,146 @@
+import * as _ from 'lodash-es';
```

Comment:
No changes to this code. I've simply moved it to a new module

Reply:
I prefer to see this as a tsx file ;)

Reply:
I agree, but I'd rather do that as a separate PR since this is existing jsx code